maintainers/maintainer-list.nix

```diff
@@ -16030,6 +16030,12 @@
     githubId = 6783654;
     name = "Nadrieril Feneanar";
   };
+  naelstrof = {
+    email = "naelstrof@gmail.com";
+    github = "naelstrof";
+    githubId = 1131571;
+    name = "naelstrof";
+  };
```

Comment:
this needs to be in a separate commit. It should also be the first commit. Otherwise LGTM.

Reply:
Separated that change out into a new commit, and ensured it comes before the butler change.